packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -149,6 +165,7 @@ class CupertinoSliverNavigationBar extends StatelessWidget {
     Key key,
     @required this.largeTitle,
     this.leading,
+    this.automaticallyImplyLeading: true,
```

Comment:
class docs also need updating

Reply:
if this can't be null, assert and document

Reply:
Done